readme.md

```diff
@@ -0,0 +1,34 @@
+# Emojo
+Lambda function to be used as a slack app to add uploaded emoji to github repo to then be deployed to slack through CI.
+
+### Config
+All config is done through environment variables. These can be set in a `.env` file in the root of the project. See `.env.example`
+
+#### `GITHUB_REPO`
+Name of the repo you wish to save your emoji too
+
+#### `GITHUB_REPO_DIR`
+A sub-directory in the repo where the images themselves will be added.
+
+#### `GITHUB_REPO_BRANCH`
```

Comment:
I think you should just auto generate a branch name. it bears no consequence really and is additional config.

also, if there are multiples open at once this will clash at present

Reply:
This is really for testing as it will be to master by default, no point creating a branch at all

Reply:
Thought we were going to create a PR so it can be approved? We could end up with shit/broken/distorted/duplicate emojis without some sort of review/holding process.

We should do at least in the interim to stop it accidentally uploading stuff as emojis.

Reply:
We can have it as an option but I'm not building this so that I can still have to go to github and merge a PR, I want FULL AUTOMATION

Reply:
Its still in git at the end of the day, so if emojo goes rogue we can still fix the repo

Reply:
But you can't fix slack. there's no delete emoji api so you can't have it clear it up. manually clicking remove. i accidentally did it to my work slack and it was very emotional trying to get rid of them all. just saying!

Reply:
We should do this to the emoji preview if not using a PR. that way you can approve straight from slack and never leave

https://api.slack.com/docs/message-attachments#attaching_buttons_to_messages

Reply:
Yea approval :+1: